zephyr/esp_shared/components/driver/dac_common.c

```diff
@@ -0,0 +1,133 @@
+/*
+ * SPDX-FileCopyrightText: 2019-2022 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+#include <zephyr/kernel.h>
+#include <esp_types.h>
+#include "esp_err.h"
+#include "esp_check.h"
+#include "driver/rtc_io.h"
+#include "driver/dac_common.h"
+#include "soc/dac_periph.h"
+#include "hal/dac_hal.h"
+
+struct {
+	unsigned int dac_spinlock;
+	uint32_t isr_core_id;
+} dac_context = {
```

Comment:
Something is not consistent in here, please check.

Reply:
I have left only dac_spinlock, thx for pointing that out

Reply:
@marekmatej, would you split this initalization call?